core/error/error_macros.h

```diff
@@ -175,7 +175,9 @@ void _physics_interpolation_warning(const char *p_function, const char *p_file,
 #define ERR_FAIL_INDEX_V(m_index, m_size, m_retval)                                                             \
 	if (unlikely((m_index) < 0 || (m_index) >= (m_size))) {                                                     \
 		_err_print_index_error(FUNCTION_STR, __FILE__, __LINE__, m_index, m_size, _STR(m_index), _STR(m_size)); \
+		GODOT_DEPRECATED_BEGIN                                                                                  \
```

Comment:
All the calls `of GODOT_DEPRECATED_*` are a bit... unfortunate, but I get it. I'm not sure I can think of a good way to avoid it.